Src/MoneyFox.Ui.Tests/Views/Statistics/StatisticCategorySummaryViewModelTest.cs

```diff
@@ -15,37 +15,43 @@ public class StatisticCategorySummaryViewModelTest
     public void StatisticCategorySummary_With_Correct_TotalExpense_TotalIncome()
     {
         // Arrange
-        var dialogService = Substitute.For<IDialogService>();
         var mediator = Substitute.For<IMediator>();
+        var dialogService = Substitute.For<IDialogService>();
         var vm = new StatisticCategorySummaryViewModel(mediator, dialogService);
+        var categorySummaries = new List<CategoryOverviewItem>
+            {
+                new CategoryOverviewItem { Value = -200 },
+                new CategoryOverviewItem { Value = -500 },
+                new CategoryOverviewItem { Value = 1000 },
+                new CategoryOverviewItem { Value = 1500 }
+            };
+        var categorySummaryModel = new CategorySummaryModel(default, default, categorySummaries);
+        mediator.Send(Arg.Any<GetCategorySummary.Query>(), Arg.Any<CancellationToken>()).Returns(x => Task.FromResult(categorySummaryModel));
 
         // Act
-        vm.CategorySummary = new ObservableCollection<CategoryOverviewViewModel>
-        {
-            new CategoryOverviewViewModel
-            {
-                CategoryId = 1,
-                Value = -200
-            },
-            new CategoryOverviewViewModel
-            {
-                CategoryId = 2,
-                Value = -500
-            },
-            new CategoryOverviewViewModel
-            {
-                CategoryId = 3,
-                Value = 1000
-            },
-            new CategoryOverviewViewModel
-            {
-                CategoryId = 4,
-                Value = 1500
-            }
-        };
+        vm.GetType().GetMethod("LoadAsync", BindingFlags.NonPublic | BindingFlags.Instance)?.Invoke(vm, null);
```

Comment:
You could here just call the LoadCommand.ExecuteAsync. That would be the public endoint that triggers this and you would not have to play around with reflection.

Reply:
@NPadrutt Made use of LoadedCommand.ExecuteAsync now.